### PR TITLE
Cobertura is not working for a simple ant project.

### PR DESCRIPTION
**Pull request by AshwiniChalla**

### CODE REVIEW

File: Tasks/ANT/ant.ps1 [hunk 1]
```diff
@@ -104,11 +104,10 @@ $summaryFile = Join-Path $buildRootPath $reportDirectoryName
 $summaryFile = Join-Path $summaryFile $summaryFileName
 # ensuring unique code coverage report task name by using guid
 $CCReportTask = "CodeCoverage_" +[guid]::NewGuid()
-# ensuring unique instrumentation task name by using guid
-$CCInstrumentTask = "CodeCoverageInstrument_" + [guid]::NewGuid()
 
 $reportBuildFileName = [guid]::NewGuid().tostring() + ".xml"
 $reportBuildFile = Join-Path $buildRootPath $reportBuildFileName
+$coberturaCCFile = Join-Path $buildRootPath "cobertura.cer"
```
Comment (allendm-msft) [inline Tasks/ANT/ant.ps1]: cant this be for any other name?

File: Tasks/ANT/ant.ps1 [hunk 2]
```diff
@@ -163,22 +168,6 @@ else
...
-		   Write-Verbose "Instrumenting classes for Code Coverage Report generation" -Verbose
-		   try
-		   {
-			   Invoke-Ant -AntBuildFile $antBuildFile -Targets $CCInstrumentTask      
-		   }
-		   catch
-		   {
-			   $instrumentationFailed = $true
-		   }   
-	   }
```
Comment (allendm-msft) [inline Tasks/ANT/ant.ps1]: i thought cobertura required instrumentation for it to work?

File: Tasks/ANT/ant.ps1 [hunk 3]
```diff
@@ -208,7 +197,7 @@ if($isCoverageEnabled)
    	}
    	else
    	{
-        	Write-Host "##vso[task.logissue type=warning;code=006003;]"
+        Write-Host "##vso[task.logissue type=warning;code=006003;]"
```
Comment (allendm-msft) [inline Tasks/ANT/ant.ps1]: indent correctly

**Comment by allendm-msft** [comment]
seems fine

**Comment by bryanmacfarlane** [comment]
accept or close

**Comment by AshwiniChalla** [comment]
This fix is already checked in.

**Pull request closed without merging**